third_party/BUILD

```diff
@@ -379,6 +379,11 @@ java_import(
     ],
 )
 
+java_import(
+    name = "flogger",
+    jars = ["flogger/flogger-0.3.1.jar"]
```

Comment:
I hate the monolithic third_party/BUILD file. Is there a reason not to use third_party/flogger/BUILD?

Also, flogger-system-backend isn't in the jars here, so it can't be used. What is the purpose?